src/Platforms/AbstractPlatform.php

```diff
@@ -426,6 +430,7 @@ public function registerDoctrineTypeMapping($dbType, $doctrineType)
      * @param string $dbType
      *
      * @return string
+     * @psalm-return non-empty-string
```

Comment:
This should also not be necessary. The `isset()` call should work fine on empty strings.

Reply:
Sorry, I don't get your point here: https://3v4l.org/6Bfiu.

Reply:
The method can return an empty string, right?

Reply:
Previously, yes. But IMO, it should not. I pushed a new commit.